torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -53,16 +59,17 @@ Message RequestCallbackImpl::processRpc(
           "size ",
           stack.size());
 
-      return std::move(ScriptResp(std::move(stack.front()))).toMessage();
+      return wrap(std::move(ScriptResp(std::move(stack.front()))).toMessage());
```

Comment:
I wonder if it would be more laconic if we do wrap in `RequestCallbackImpl::processMessage` instead of in every switch-case here?

Reply:
- The problem is that we can only set the messageId() (as wrap does) once the future is completed. Creating an additional future hop for this feels suboptimal. This was the least verbose way I thought of doing for the non-async cases.

- I don't think I remember using "laconic" since I was 17 and studying for the SATs :)